tests/test_averager2d.py

```diff
@@ -163,7 +163,7 @@ class AligningAverager2DBase(Averager2DBase):
     """
 
     averager = AligningAverager2D
-    num_procs = 1  # paralleized subclasses may override
+    num_procs = 1 if xfail_ray_dev() else 2
```

Comment:
Am I wrong in thinking this will never activate the parallelization, because of the `num_procs < 3 ==> num_procs = 1` condition in `Averager2D`?

Reply:
Logic is below. When an integer is explicitly provided the code will attempt to use that integer (after checking it is >0 ).  The < 3 disable is for the "automatic" config that happens when user provides the `None` default.

```
        if num_procs is None:
            num_procs = num_procs_suggestion()
            # Only enable multiprocessing when several cores available                                     
            if num_procs < 3:
                num_procs = 1
        elif not (isinstance(num_procs, int) and num_procs > 0):
            raise ValueError(
                f"num_procs should be a positive integer, passed {num_procs}."
            )
        self.num_procs = num_procs
```

Reply:
Okay makes sense thanks